### PR TITLE
Monitor promo pricing

### LINKED ISSUE #2046: BatchExplorer does not report pricing on promo SKU's
**To Reproduce**
Hard to reproduce as you need to be whitelisted to the sku to deploy

**Expected behavior**
Pricing similar to other skus

**Screenshots**
![image](https://user-images.githubusercontent.com/18502617/59041675-efd30280-882d-11e9-

### PR DESCRIPTION
**Pull request by bgklein**
image

Fixes #2046

### CODE REVIEW

File: src/app/services/pricing.service.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { log } from "@batch-flask/utils";
 import { ArmBatchAccount, BatchSoftwareLicense, Pool, RateCardMeter } from "app/models";
 import { BatchPricing, OSPricing, OsType, SoftwarePricing, VMPrices } from "app/services/pricing";
 import { PoolPrice, PoolPriceOptions, PoolUtils } from "app/utils";
-import { DateTime } from "luxon";
```
Comment (timotheeguerin) [inline src/app/services/pricing.service.ts]: remove?

File: src/app/services/pricing/pricing.model.ts [hunk 2]
```diff
@@ -122,7 +122,8 @@ export class NodePricing {
             lowpri = true;
             name = name.replace(" Low Priority", "");
         }
-        const vmSizes = name.split("/").map(x => x.replace(/ /g, "_"));
+        const vmSizes = name.split("/").map(x => x.replace(/ /g, "_").concat(
+            category.contains("Promo") ? "_Promo" : ""));
```
[the remark this answers is not in the document]
Comment (timotheeguerin) [reply src/app/services/pricing/pricing.model.ts]: I think it lowercase everything later in the pipeline anyway though

[the remark this answers is not in the document]
Comment (bgklein) [reply src/app/services/pricing/pricing.model.ts]: Its Promo when referring to the actual Id, but later on you end tolower'ing everything for comparisons cause nobody is consistent ;)  I beleive at this point the rest of the VmSize uses Standard_NC6 type naming still

[the remark this answers is not in the document]
Comment (timotheeguerin) [reply src/app/services/pricing/pricing.model.ts]: alright sounds good

**Pull request merged**